File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/FFI-4/cpp/ffi4.cpp
#include <cstdio>
#include "supplied.h"   /* declarations consumed mechanically; none hand-written */
int main() {
    SuppliedRecord r; r.count = 3; r.weight = 1.5;
    std::printf("supplied_add=%d\n", supplied_add(20, 22));
    std::printf("supplied_weighted=%.2f\n", supplied_weighted(&r));
    std::printf("SUPPLIED_SCALE=%d\n", SUPPLIED_SCALE);
    return 0;
}

File: benchmark/2026-09-17-7677581(latest)/standard/rubrics/probes/FFI-4/swift/ffi4.swift
import Foundation   // re-exports Darwin; the C standard library reaches Swift through first-party C interop
// Declarations come from probes/FFI-4/c/supplied.h via -import-objc-header; none are hand-written here.
var r = SuppliedRecord(count: 3, weight: 1.5)
print("supplied_add=\(supplied_add(20, 22))")
print(String(format: "supplied_weighted=%.2f", supplied_weighted(&r)))
print("SUPPLIED_SCALE=\(SUPPLIED_SCALE)")
